[AGTMETRICS-393] Add mention of Agent Data Plane and preview intake form. (#29636)
DataDog/documentation: content/en/developers/dogstatsd/high_throughput.md

Most of the time the symptoms can be alleviated by tweaking some configuration options described below.

> <div class="alert alert-info">
>   <b>Working with a high volume of custom metrics?</b>
>   
>   <div>
>     <b>Agent Data Plane</b> is a new Agent feature for processing high-volume custom metrics workloads with improved CPU and memory efficiency. This feature is in Preview. If you are interested in trying Agent Data Plane, reach out by filling out <a href="https://fill-me-out">this form</a>.
>   </div>
> </div>
> 

Reviewer: ```suggestion
<div class="alert alert-info">
Agent Data Plane is a Preview feature for processing high-volume custom metrics workloads with improved CPU and memory efficiency. You can <a href="#">request access</a> to this Preview feature.
</div>
```

I edited this copy down a bit — basically, to maintain public trust in the accuracy and objectivity of documentation, we have to diminish any appearance of marketing or marketing-adjacent behavior. We can't really _sell_ a Preview feature, but we can (as dispassionately and objectively as possible) affirm its existence. 

Also, this still needs a link to the preview form!

Author: Yeah, no, that totally makes sense and I appreciate the copyediting. 👍🏻 

I should be getting the link updated soon as we were waiting on getting the intake form created.
